website/content/api-docs/system/internal-counters.mdx

```diff
@@ -1133,7 +1133,7 @@ provided start and end times. The returned set of client information will be
 deduplicated over the time window and will show the earliest activity logged for
 each client. The output will be ordered chronologically by month of activity.
 
-<Note title="Techn preview">
+<Note title="Tech preview">
```

Comment:
This is the main change, the rest were just automatic formatting fixes (e.g. incorrectly formatted table)